images/linux/post-generation/systemd-linger.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Enable user session on boot, not on login
+loginctl enable-linger 1001
```

Comment:
We should get user id like we do - https://github.com/actions/virtual-environments/blob/main/images/linux/post-generation/environment-variables.sh